aim/cli/repo/commands.py

```diff
@@ -0,0 +1,56 @@
+import click
+import os
+
+from aim import Run
+from aim.cli.runs.utils import list_repo_runs
+from aim.sdk.repo import Repo
+from aim.sdk.utils import clean_repo_path
+
+
+@click.group()
+@click.option(
+    '--repo',
+    required=False,
+    default=os.getcwd(),
+    help='Path to the aim repository.',
+    type=click.Path(exists=True, file_okay=False, dir_okay=True, writable=True),
+)
+@click.pass_context
+def repo(ctx, repo):
```

Comment:
do we need command/sub-command setup here?
`aim repo --repo <REPO_PATH> cleanup` seems duplicating things and a bit confusing

Reply:
I renamed it to `--path`